service_storagefacility/src/main/scala/no/uio/musit/microservice/storagefacility/dao/storage/KdReportDao.scala

```diff
@@ -0,0 +1,108 @@
+package no.uio.musit.microservice.storagefacility.dao.storage
+
+import com.google.inject.Inject
+import no.uio.musit.service.MusitResults.{ MusitDbError, MusitResult, MusitSuccess }
+import play.api.db.slick.{ DatabaseConfigProvider, HasDatabaseConfigProvider }
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
+import slick.driver.JdbcProfile
+
+import scala.concurrent.Future
+
+/**
+ * Created by ellenjo on 26.09.16.
+ */
+
+class KdReportDao @Inject() (val dbConfigProvider: DatabaseConfigProvider) extends HasDatabaseConfigProvider[JdbcProfile] {
+
+  import driver.api._
+
+  def getReportTotalArea: Future[MusitResult[Double]] = {
+    db.run(
+      sql"""
+         select SUM(area_to)
+         from MUSARK_STORAGE.STORAGE_NODE
+         WHERE storage_type = 'Room'
+      """.as[Double].head.map(MusitSuccess.apply)
+    ).recover {
+        case e: Exception => MusitDbError("Error occurred while retrieving objects", Some(e))
+      }
+  }
+
+  /*
+perimeterSecurity: Int,
```

Comment:
FIXME
This comment is an excellent example of something that should be placed in the scaladoc of the `class` definition. Or even better just reference the .sql file that contains the table definition. If the table changes, this comment will become invalid very easily.